AdAway/src/main/res/values/strings_preferences.xml

```diff
@@ -46,7 +46,7 @@
         <item>customTarget</item>
     </string-array>
 
-    <string name="pref_apply_method_def" translate="false">writeToSystem</string>
+    <string name="pref_apply_method_def" format="string" type="string"">writeToSystem</string>
```

Comment:
This string should not be translated.
It will cause trouble with `org.adaway.helper.ApplyHelper` and `org.adaway.helper.RevertHelper` otherwise.

Reply:
@PerfectSlayer I agree, none of these are translatable to be exact. 
However, I studied the structure of the xml a bit and this is just a correction of an inconsistency I recognized and does not change the behavior of translation tools.
See, all `_key` tags (should) have the `translate=false` attribute and all `_def` tags (should) have `format` & `type` attributes. Even though none of the `_def` tags have the `translate` attribute, those with these two attributes were not translated by the translation tools so far. 
```xml
<string name="pref_apply_method_def" >
``` 
had the `translate` attribute by mistake I guess, so I corrected it by removing it and adding the `format` & `type` attributes instead.